### PR TITLE
Obi 605 Keyboard usage

### PR DESCRIPTION
**Pull request by MelanieJakobArbeit**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: src/components/app/NavigationBar.tsx [hunk 1]
```diff
@@ -123,9 +129,57 @@ export const NavigationBar = ({
 		return value ? `walkthrough-${value}` : '';
 	}, []);
 
+	const handleKeyDownMenu = (e, index) => {
+		if (e.key === 'Enter' || e.key === ' ') {
+			if (document.activeElement === ref_logout.current) {
```
Comment (alxbenz) [inline src/components/app/NavigationBar.tsx]: Discussion: Could we improve this section by using switch case?

File: src/components/select/SelectDropdown.tsx [hunk 2]
```diff
@@ -371,6 +373,19 @@ export const SelectDropdown = (props: SelectDropdownItem) => {
...
+				ref={props.selectRef}
+				openMenuOnFocus={props.isInsideMenu ? true : false}
+				closeMenuOnSelect={true}
+				onMenuClose={() => {
+					if (props.isInsideMenu) {
+						setTimeout(() => {
+							document
+								.getElementById('local-switch-wrapper')
+								.focus();
+						}, 10); //we need this timeout because the menu is not closed when switching the focus
```
Comment (alxbenz) [inline src/components/select/SelectDropdown.tsx:386]: 👍

File: src/components/select/select.styles.scss [hunk 3]
```diff
@@ -22,14 +22,15 @@ $select-menu-before-display: block !default;
...
+			outline: $focus-outline;
+			outline-offset: 4px;
+			border-radius: 4px;
+			transition: all 0.6s ease;
 		}
 
-		&:focus {
-			outline: none;
-		}
+		// &:focus {
```
Comment (alxbenz) [inline src/components/select/select.styles.scss]: please remove if not needed anymore

File: src/components/sessionsList/SessionsList.tsx [hunk 4]
```diff
@@ -727,18 +727,66 @@ export const SessionsList = ({
 		[type, userData]
 	);
 
+	const ref_tab_first = useRef<any>();
+	const ref_tab_second = useRef<any>();
+	const ref_list_array = useRef<any>([]);
+
+	const handleKeyDownTabs = (e) => {
+		console.log('e', e);
```
Comment (alxbenz) [inline src/components/sessionsList/SessionsList.tsx]: please remove

**Review by alxbenz — CHANGES_REQUESTED** [review]

File: src/components/app/NavigationBar.tsx (hunk 1 above)
Comment (MelanieJakobArbeit) [reply src/components/app/NavigationBar.tsx]: yes we can

**Review by alxbenz — DISMISSED** [review]

File: src/components/app/NavigationBar.tsx [hunk 5]
```diff
@@ -123,9 +129,60 @@ export const NavigationBar = ({
 		return value ? `walkthrough-${value}` : '';
 	}, []);
 
+	const handleKeyDownMenu = (e, index) => {
+		switch (e.key) {
+			case 'Enter':
+			case ' ':
+				if (document.activeElement === ref_logout.current) {
```
Comment (alxbenz) [inline src/components/app/NavigationBar.tsx]: Better. But still really complex... (SonarCloud thinks so as well).
You could extract the conditions into variables and maybe move each case into a separate function?
Not sure. What do you think?

Comment (MelanieJakobArbeit) [reply src/components/app/NavigationBar.tsx]: Yes that could help. I'll try it out.

**Review by alxbenz — APPROVED** [review]

**Pull request merged**